index.html

```diff
@@ -1,220 +1,218 @@
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="./style.css" type="text/css">
-    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
-    <title>portfolio</title>
-  </head>
-  <body>
-    <!-- first section -->
 
-    <div class="card-container">
-      <header>
-        <nav class="flex-nav">
-          <a href="" class="logo">chester</a>
-          <a href="" class="union-icon"><img src="./images/Union.png" alt=""></a>
-        </nav>
-      </header>
-      <section class="headline-section">
-        <div class="text-area">
-          <h1>I'm rotshidzwa <br>Glad to see <br>you!</h1>
-          <p>I’m a software developer! I can help you build a product, feature or website Look through some of my work and experience!
-            If you like what you see and have a project you need coded, don’t hestiate to contact me.
-          </p>
-        </div>
-        <ul class="media-icon ">
-            <li>
-                <a href="# " class="icons ">
-                    <img src="./images/twitter.png " alt="twitter ">
-                </a>
-            </li>
-            <li>
-                <a href="# " class="icons ">
-                    <img src="./images/Linkedin-icon.png " alt="linkedin ">
-                </a>
-            </li>
-            <li>
-                <a href="# " class="icons ">
-                    <img src="./images/medium.png " alt="medium ">
-                </a>
-            </li>
-            <li>
-                <a href="# " class="icons ">
-                    <img src="./images/git.png " alt="git ">
-                </a>
-            </li>
-            <li>
-                <a href="# " class="icons ">
-                    <img src="./images/engelist.png " alt="angelist ">
-                </a>
-            </li>
-        </ul>
+<html  lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <link rel="stylesheet" href="style.css">
+        <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap" rel="stylesheet">
+        <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <script src="https://kit.fontawesome.com/1dea6ac3ad.js" crossorigin="anonymous"></script>
+        <title>Portfolio</title>
+    </head>
+    <body>
+        <section class="Section-1">
+        <header id="Header"> 
+            <h3>  <a href="#" id="logo" class="desi">Chester</a></h3> 
+               <nav>
+                <ul class="desktop desi">
+                    <li class="nev-list">About</li>
+                    <li class="nev-list">Portfolio</li>
+                    <li class="nev-list">Contact</li>
+                </ul>
+                <i class="fa-solid fa-bars" id="Menu_id"></i>
+               </nav>
+        </header>
```

Comment:
- [ ] Kindly remove the extra space on the top of the header section in the desktop view, to match the Figma design.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/161293391-00fe75b0-aeb6-4a88-b219-f533306ed22b.png) | ![image](https://user-images.githubusercontent.com/62290545/161293580-abddbbcf-008c-4f80-8618-6881c149b3b5.png) |